storybook/stories/tokens/typography.mdx

```diff
@@ -0,0 +1,178 @@
+import { Meta, Typeset } from '@storybook/addon-docs/blocks';
+
+<Meta title="Tokens/Typography" name="page" />
+ 
+export const typography = {
+	type: {
+		primary: '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif',
+	},
+	weight: {
+		regular: '400',
+		medium: '500',
+	},
+	size: {
+		s1: 11,
+		s2: 12,
+		s3: 13,
+		s4: 15,
+		s5: 20,
+		s6: 32,
+	},
+};
+ 
+export const SampleTextHeading = 'Code is Poetry.';
+export const SampleTextParagraph = 'WordPress grows when people like you tell their friends about it, and the thousands of businesses and services that are built on and around WordPress share that fact with their users.'
+ 
+# Typography tokens
+
+This document outlines the various tokens relating to typography in the WordPress components system.
+
+## Semantic tokens
+
+Semantic tokens compose primitive tokens to create reusable type styles enhancing consistency across the software.
+
+They are defined as sass mixins and can be used like so:
+
+```
```

Comment:
We can get some syntax highlighting if we specifically mark this code block as a CSS one.

```suggestion
```css
```

Reply:
Since this is SASS, should try this instead?

```suggestion
```sass
```
or maybe 

```suggestion
```scss
```

Not sure if either of them is supported

Reply:
The same would apply to other SASS code snippets

Reply:
None of those 2 are supported (I tried them out while reviewing the PR), which is why I suggested `css`.

Reply:
Thank you for clarifying!